status/flowcells.py

```diff
@@ -40,6 +41,26 @@ def formatDate(date):
     datestr=datetime.datetime.strptime(date, "%Y-%m-%d %H:%M:%S.%f")
     return datestr.strftime("%a %b %d %Y, %I:%M:%S %p")
 
+
+def find_id(stringable, pattern_type: str) -> re.match:
+
+    string = str(stringable)
+
+    patterns = {
+        "project": re.compile("P\d{5}"),
+        "sample": re.compile("P\d{5}_\d{3}"),
```

Comment:
Throw in possibility of 6 digit projects here as well and we'll be safer for the future.

Reply:
[93b885a](https://github.com/SciLifeLab/genomics-status/pull/740/commits/93b885adb1c6aa6b7eccb8759d5b8da9a4745a6d)